### ISSUE TITLE
Grid filter row is not locked.

### ISSUE THREAD

**Issue by simonssspirit** [telerik/kendo-react#191]
Grid filter row is not locked.

If the Grid has locked columns the filter row is not locked:

https://stackblitz.com/edit/react-wrurgl?file=app%2Fmain.js

**Comment by fabio-b** [comment]
I also want to note that if you add a custom cell component it does not lock as well.

In this example, ProductID and Discontinued columns are locked. ProductID has a custom Cell component, the header is locked but the corresponding cell is not. Discontinued is displaying correctly.
https://mkz38x.run.stackblitz.io/

**Comment by Xizario** [comment]
@fabio-b , the custom cell should return `td` element, not `div`, and apply the settings that comes from the props, such as style. Here is modified version that will work for your case:
```jsx
class Cell extends React.Component {
  render() {
    return (
      <td {...this.props}>
        <div>{this.props.value}</div>
      </td>
    )
  }
}
```

**Comment by fabio-b** [comment]
Thank you @Xizario! That works. I would just suggest adding this to the documentation under custom Cell rendering as it's missing this bit of detail. Thanks again.

**Comment by Xizario** [comment]
Fixed in `2.8.0`
